build.ps1

```diff
@@ -311,11 +311,11 @@ function Nupkg
             #Only show package analysis for newer packages
             if($IsNetCoreBuild)
             {
-                . $nuget pack $filePath -Version $Version -OutputDirectory $NugetPackagePath -Properties "RedistVersion=$RedistVersion;"
+                . $nuget pack $filePath -Version $Version -OutputDirectory $NugetPackagePath -Properties "RedistVersion=$RedistVersion;Branch=$env:APPVEYOR_REPO_BRANCH;CommitSha=$env:APPVEYOR_REPO_COMMIT;"
```

Comment:
The nuget pack is ok with these being empty? e.g. for local builds?

Reply:
Yes, nuget pack is ok with that. But we can get the actual branch / commit by calling git itself so it will fill it locally too.

Reply:
Take a look at my comment here: https://github.com/cefsharp/CefSharp/pull/4763#pullrequestreview-1941880185

The easier solution should be to use source link which will handle it.

Reply:
@thompson-tomo We are using `nuget pack` to create our packages via hand written `nuspec` files. There are arch specific builds for `x86/x64/arm64` that complicate things.

We already have source link enabled on the projects themselves. Far as I'm aware source link reads from the pdb files, it doesn't actually take any reference to the nupkg file.

Reply:
> Yes, nuget pack is ok with that. But we can get the actual branch / commit by calling git itself so it will fill it locally too.

@campersau Awesome! Thank you!